packages/react-api/__tests__/hooks/useCartoLayerProps.test.js

```diff
@@ -23,6 +23,7 @@ describe('useCartoLayerProps', () => {
       'connection',
       'credentials',
       'clientId',
+      'queryParameters',
```

Comment:
If we add something to public api, we should prepare also a small PR at doc repo, so we have this piece updated here: https://docs.carto.com/react/library-reference/api/.

We haven't decided yet if this is gonna be released in alpha mode or as patch.